server/lobbyconnection.py

```diff
@@ -665,6 +708,20 @@ async def command_hello(self, message):
 
         await self.send_game_list()
 
+    async def update_irc_password(self, conn, login, password):
+        passwordmd5 = md5(password.encode()).hexdigest()
+        # Since the password is hashed on the client, what we get at this point
+        # is really md5(md5(sha256(password))). This is entirely insane.
```

Comment:
:facepalm: nono! :-D

Reply:
From a security perspective  applying hash algorithms on hashes is bad